src/main/target/common_pre.h

```diff
@@ -124,6 +124,10 @@
 #define SRAM2
 #endif
 
+#if defined(STM32F4) || defined(STM32F7)
```

Comment:
Probably more consistent if we add this to the sections for STM32F4 and STM32F7 above.

Reply:
No, I meant the two `#ifdef STM32F4` and `#ifdef STM32F7` blocks above that (https://github.com/betaflight/betaflight/pull/6837/files#diff-016d7c2e306c7201ea0ec246bc617247R50). The scheduler related constant pair is different as it needs an `#else` case, whereas other conditionals that are specific to MCU type should go into the blocks above in my opinion.

Reply:
Got it 👍